### DISCUSSION TITLE
Fingerprint Emulation ask

### DISCUSSION THREAD

**Discussion by apuromafo** [Genymobile/feedback#72] [Genymotion Desktop] [answered]
The Biometric Authentication
context: i can buy a device usb, there can read the fingerprint, but need know if really can be useful to use with genymotion, can helpme?
My license: (business).
i read there android avd have fingerprint 
adb command have some for finger.
usb maybe with otg in normal phone can be useful..
But i need know if are something compatible to use, for request to test biometric.

Br, Michel F.

**Comment by bhein-sherpa6** [comment]
Fingerprint emulation is not currently available with Genymotion, but is apparently on the roadmap. Please see this discussion: https://github.com/orgs/Genymobile/discussions/59#discussion-5331426

**Comment by agavignet** [comment]
Little update: this feature is currently in development on our side, should be available before the end of the year but most probably in november.

**Comment by agavignet** [comment, marked as answer]
Hi @apuromafo,

Desktop 3.6 has been released and brings a new Biometrics widget, it might help you a lot :)
Check out our blog post https://www.genymotion.com/blog/genymotion-desktop-3-6/ 

Adrien
